### PR TITLE
Ensure that the admin client jsonnet has correct S3 bucket property.

### PR DESCRIPTION
**Pull request by hedss**
Signed-off-by: Heds Simons <hedley.simons@grafana.com>

**What this PR does**:
Fix the property name for the admin client configuration, changing it to `bucket_name` from the errorneous `bucket`.

**Which issue(s) this PR fixes**:
N/A - PR without Issue.

**Checklist**
- [ ] Tests updated
- [ ] Documentation added
- [x] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: operations/jsonnet/enterprise/main.libsonnet [hunk 1]
```diff
@@ -64,7 +64,7 @@ tempo {
       storage: {
         type: $._config.backend,
         s3: {
-          bucket: $._config.bucket + '-admin',
+          bucket_name $._config.bucket + '-admin',
```
Comment (annanay25) [inline operations/jsonnet/enterprise/main.libsonnet:67]: are we missing a colon here?

**Comment by annanay25** [comment]
Thanks Heds, also please sign the CLA! :)

File: operations/jsonnet/enterprise/main.libsonnet (hunk 1 above)
Comment (hedss) [reply operations/jsonnet/enterprise/main.libsonnet]: Fix the property and miss out the colon 🤦 Must have stuffed up whilst commiting, thanks Annanay. :)
Have also signed the CLA!

**Review by annanay25 — APPROVED** [review]
Thanks!

**Pull request merged**